index.html

```diff
@@ -1,15 +1,18 @@
+
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>whichcar</title>
-        <script src="https://cdn.tailwindcss.com"></script>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link rel="icon" href="img/icon.svg" class="w-10 h-10 text-white p-1/2 bg-blue-500 rounded-full">
-    </head>
+<head>
+    <title>whichcar</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="img/icon.svg" class="w-10 h-10 text-white p-1/2 bg-blue-500 rounded-full">
+    <link rel="stylesheet" href="styles/app.css">
+
+</head>

```

Comment:
(nit): Can we Format the file, it'll resolve the indentation issues reflecting as diff

Reply:
No problem.